src/ProjectTest.java

```diff
@@ -0,0 +1,57 @@
+import junit.framework.*;
+
+/**
+ * Project-wide base class for Testing.
+ */
+public class ProjectTest extends TestCase {
+
+    /**
+     * Constructor.
+     */
+    public ProjectTest(String name) {
+        super(name);
+    }
+
+    /**
+     * Assert if two Quaternions are equal within a tolerance.
+     *
+     * @param expected the expected Quaternion.
+     * @param actual   the actual Quaternion.
+     * @param delta    the tolerance.
+     */
+    public void assertEqualsQuaternion(Quaternion expected, Quaternion actual, double delta) {
```

Comment:
No need to change it, but just FYI you can write your custom assertions: https://ccguyka.wordpress.com/2015/02/08/custom-junit-assertions/